docs/guides/v7-upgrade-guide.md

```diff
@@ -251,23 +233,24 @@ import { Table } from '@instructure/ui-table'
 
 DateInput has been removed from ui-forms; use DateInput from `ui-date-input` instead. DateInput is a controlled only component and will need to be composed in the consuming app. View the [component documentation](#DateInput) for examples of managing state for the updated DateInput.
 
-
 <!-- ////////////////////////////////////////////// -->
 <!-- THIS IS WHERE THE DATE/TIME INPUT STUFF STARTS -->
 <!-- ////////////////////////////////////////////// -->
 
 ### DateTimeInput
-DateTimeInput has been removed from ui-forms. See a __read only__ example of implementing a [DateTime Input Pattern](/#TimeDate) using Select and DateInput.
 
+DateTimeInput has been removed from ui-forms. See a **read only** example of implementing a [DateTime Input Pattern](/#TimeDate) using Select and DateInput.
```

Comment:
I saw somewhere above that you mentioned it's been brought back in V8. Should we put an "update: " block here too?